src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -132,8 +132,29 @@ trait IterableOps[+A] extends Any {
   }
 
   override def toString = s"$className(${mkString(", ")})"
+
+  def sameElements[B >: A](that: Iterable[B]): Boolean = {
```

Comment:
See also: https://groups.google.com/forum/#!msg/scala-debate/je4vZbR3WbA/_fa5aNyTyuoJ, the method name is just wrong, I hope now is the chance to fix it.